NullAsserts.cs

```diff
@@ -25,7 +28,7 @@ public static void NotNull(object @object)
         /// </summary>
         /// <param name="object">The object to be inspected</param>
         /// <exception cref="NullException">Thrown when the object reference is not null</exception>
-        public static void Null(object @object)
+        public static void Null(object? @object)
```

Comment:
💡 This should use `[MaybeNull]` to ensure subsequent code treats the value as null even if it was previously inferred to be non-null (or oblivious).

```suggestion
        public static void Null([MaybeNull] object? @object)
```

Reply:
Good find, thanks Sam 👍